roles/smoker/tasks/main.yml

```
@@ -4,6 +4,25 @@
   package:
     name: '{{ smoker_browser_packages }}'
 
+- name: 'Install podman'
+  become: true
+  package:
+    name: 'podman'
+
+- name: Set max user namespaces
+  become: true
+  sysctl:
+    name: user.max_user_namespaces
+    value: 100000
+
+- name: Set subuid range for user
+  become: true
+  command: usermod --add-subuids 10000-75535 {{ ansible_user }}
+
+- name: Set subuid range for user
```

This doesn't match the content
```suggestion
- name: Set subgid range for user
```